src/python/WMComponent/JobCreator/JobCreatorPoller.py

```diff
@@ -563,6 +571,13 @@ def pollSubscriptions(self):
 
 
                 # Assemble a dict of all the info
+                inputDataset = None
+                locations = None
+                dbsurl = wmTask.inputDatasetDBSURL()
```

Comment:
I don't recall now why the code here is as it is. Your right that the changes were related to the changes in Condor. When you suggest changing the string, to what should it be changed, the dbsurl used in production? I assume that the wmTask in unittest has a dummy string in order to decouple from actually using the production url right?